sos/plugins/systool.py

```diff
@@ -0,0 +1,43 @@
+# systool.py
+# Copyright (C) 2007-2014 Red Hat, Inc., Jon Magrini <jmagrini@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+import os
+import os.path
+from sos.plugins import Plugin, RedHatPlugin, os
+
+class sysfsutils(Plugin, RedHatPlugin):
+    """View system device information by bus, class, and topology
+    """
+
+    plugin_name = 'sysfsutils'
+    profiles = 'storage'
+    packages = 'sysfsutils'
```

Comment:
Stylistically `packages` is usually a tuple, e.g. `packages = ('sysfsutils', )`. A string works here because we convert strings to a list internally, however only a few plugins use a list here and no other plugins use a string currently (anymore?).

Reply:
Plugin class names (other than derived classes with a tagging class prefix, e.g., `RedHatFooPlugin`) must match the filename that contains the. It's either `systool` or `sysfsutils`, but not both.